src/lib/utils/thorchain/lp.ts

```diff
@@ -233,34 +213,23 @@ export const estimateRemoveThorchainLiquidityPosition = async ({
 }
 
 export const calculateEarnings = (
-  _assetLiquidityFees: string,
-  _runeLiquidityFees: string,
+  pool: MidgardEarningsHistoryPoolItem,
   userPoolShare: string,
   runePrice: string,
```

Comment:
Can we make this `runePriceUserCurrency` here to make it clear this isn't in USD?